src/components/bcros/filing/item/StaffFiling.vue

```diff
@@ -0,0 +1,77 @@
+1
+<template>
+  <BcrosFilingCommonTemplate :filing="filing" data-cy="staff-filing">
+    <template #title>
+      <UIcon v-if="isTypeCourtOrder" name="i-mdi-gavel" />
+      <!--  todo: should we internationalize this ?? -->
```

Comment:
As with the other one my instinct on this is we should if we're translating the text (text.filing.filed)